### PR TITLE
fix(indexer) skip ovm 1.0 hashes

### PR DESCRIPTION
**Pull request by hamdiallam**
Hardcode a list of withdrawals to OVM1 withdrawals to skip when finalizing bridge events. We're going with hardcoding it since the list isn't large enough to desire a different solution.

The CSV file and dune query used to create this list is provided in the repository for reference (not embedded in the go binary).

Dune Query for the retrieved hashes: https://dune.com/queries/3231132

### CODE REVIEW

**Comment by hamdiallam** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #8046** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8046?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #8093** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8093?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #8087** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8087?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/8087?utm_source=stack-comment).

File: indexer/processors/bridge/l1_bridge_processor.go [hunk 1]
```diff
@@ -155,12 +156,25 @@ func L1ProcessFinalizedBridgeEvents(log log.Logger, db *database.DB, metrics L1M
...
+	skippedOVM1Withdrawals := 0
 	for i := range provenWithdrawals {
 		proven := provenWithdrawals[i]
 		withdrawal, err := db.BridgeTransactions.L2TransactionWithdrawal(proven.WithdrawalHash)
 		if err != nil {
 			return err
 		} else if withdrawal == nil {
+			// Handle networks with OVM 1.0 state (OP Mainnet/Goerli) in order to gracefully skip these withdrawals
+			portalAddr := l1Contracts.OptimismPortalProxy
+			if portalAddr == config.Presets[10].ChainConfig.L1Contracts.OptimismPortalProxy || portalAddr == config.Presets[420].ChainConfig.L1Contracts.OptimismPortalProxy {
```
Comment (roninjin10) [inline indexer/processors/bridge/l1_bridge_processor.go]: I don't think goerli has ovm 1.0 state. I think we were using Kovan way back then

**Review by roninjin10 — APPROVED** [review]

File: indexer/processors/bridge/l1_bridge_processor.go (hunk 1 above)
Comment (hamdiallam) [reply indexer/processors/bridge/l1_bridge_processor.go:168]: ohh okay interesting, i'll just leave it in a sanity. The cross domain message nonce also starts at 100k for goerli so not harmful to include just incase

First message sent from OP-Goerli: https://goerli-optimism.etherscan.io/tx/0x254d9c28add020404142f840ed794cea51f86c0f0a737e3e7bdd7e1e4550962e#eventlog

Comment (hamdiallam) [reply indexer/processors/bridge/l1_bridge_processor.go]: yea you're right, i checked the Goerli genesis timestamp in comparison with the regenesis state. Georli should just be OVM 2.0

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**